spec/food_order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'food_order'
+
+describe FoodOrder do
+
```

Comment:
Because the class `FoodOrder` is depending on `Menu`, it should be isolated from the Menu class in those unit tests (using a double, for example). To do this you'll need to:
 * inject the `@menu` attribute in the initialize method, rather than instantiating directly the Menu class in it: `@menu = Menu.new`
 * explicitly define the RSpec's subject so you can create the instance of `FoodOrder` and pass the Menu double in it
 * you'll know if your class is isolated correctly if you can "break" the `Menu` class (maybe by commenting out its method), but the tests for `FoodOrder` still pass